src/app/dialogs/export-experiment/export-experiment.component.html

```diff
@@ -15,7 +42,7 @@ <h1 mat-dialog-title>Export Experiment</h1>
 </div>
 <div class="dialog-actions" mat-dialog-actions>
     <button mat-button (click)="onCancel()">Cancel</button>
-    <button mat-button [mat-dialog-close]="{test: configTest}">
-        Export Experiment
-    </button>
-</div>
\ No newline at end of file
+    <button mat-button [disabled]="downloading || this.downloadLink" (click)="onExport()">Export Experiment</button>
+    <a role="button" mat-button [disabled]="!this.downloadLink" href="{{this.downloadLink}}"
```

Comment:
It is better not to show the download link in an easily dismissable dialog, as it blocks the UI until the download is available.